### DISCUSSION TITLE
I think you might won't update gt4 series to zepp os 3.0

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#149] [General]
Because previously too you guys didn't updated the GT3 to zepp os 2.0 . So I am scared that you guys might won't put 3.0 to GT4 series. I think you guys will launch GT5 series which will support 3.0.

**Comment by vaas1993** [comment]
I share your sentiment. In fact, they have not been making significant efforts in improving their GT4 system, with many users reporting numerous bugs and flawed interaction logic within the mobile app. While prioritizing profit is expected for a commercial company, as a consumer, I am also disappointed. Perhaps I will no longer consider this brand in the future, nor recommend it to others around me.

**Reply by alexmathew11** [reply]
Yeah I also think so. Let's see what GT5 series have got. Because I expect to be the biggest upgrade. One thing which disappoints me the most about amazfit is that when ever they launch a new wearable they stop updating there old generation models. And even if they do they just fix the bugs ,they don't really add new features . If they could just make all zepp os version upto 4 or 5 support to amazfit gt4 series it would be a complete package. Let's hope so

**Reply by MyoungXUE** [reply]
The Zepp OS team has been taking developer feedback very seriously, working on bug fixes and new features.

We have just released Zepp OS 2.1 features, and in the future, there will be models that can be upgraded to Zepp OS 2.1.

For details, please refer to https://docs.zepp.com/docs/guides/version-info/new-features-21/

Upgrading older devices takes up a lot of R&D resources, and some devices cannot be upgraded to the newer system due to hardware performance limitations, so we will do our best to get more models upgraded.

**Reply by alexmathew11** [reply]
I am so much proud of amazfit that they actually deal with the criticism very positively. And for adding new features and functions let sit back and see what amazfit have got for us .
